modules/custom_operations/user_ie_extensions/sparse_conv_transpose.cpp

```diff
@@ -0,0 +1,109 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "sparse_conv_transpose.hpp"
+
+using namespace TemplateExtension;
+
+//! [op:ctor]
+SparseConvTranspose::SparseConvTranspose(const ov::Output<ov::Node>& features,
+                                         const ov::Output<ov::Node>& inp_pos,
+                                         const ov::Output<ov::Node>& out_pos,
+                                         const ov::Output<ov::Node>& kernel,
+                                         const ov::Output<ov::Node>& offset) : Op({features, inp_pos, out_pos, kernel, offset}) {
+    constructor_validate_and_infer_types();
+}
```

Comment:
For operations that we are not using manually in the code (with `std::make_shared<OpName>(params)`, we can save the number of code lines by defining ctor with vector of inputs and then just forward it to Op ctor. Then it will simplify the code in `clone_with_new_inputs`. Creating ctors with multiple arguments handy in case if you are manually building the graph of a model. Here we are not doing it.

Reply:
Thanks, done